README.md

```diff
@@ -77,12 +110,59 @@ To start the daemon, run:
 opfgd start --cfg config.toml
 ```
 
-### Running tests
+## Querying Block Finality Status
+
+The finality gadget provides gRPC endpoints to query the finalization status of blocks. You can use `grpcurl` to test these endpoints.
+
```

Comment:
This gadget also supports HTTP / JSON-RPC. We could probably document that as well https://github.com/babylonlabs-io/rollup-finality-gadget/blob/main/server/httpserver.go

Reply:
Opened issue for it https://github.com/babylonlabs-io/rollup-finality-gadget/issues/53

Reply:
Sounds good. Actually the HTTP server might be more widely used in practice, since it can be used as a proxy for the ETH RPC endpoints (like what Blitz is doing). Can be done in a separate PR for sure